sandbox-backend/src/main/java/com/exadel/sandbox/team5/entity/City.java

```diff
@@ -0,0 +1,29 @@
+package com.exadel.sandbox.team5.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+
+@Entity
+@Table(name = "city")
+public class City extends BaseEntity{
+
+    @Column(name = "name")
+    private String name;
+
+    @OneToMany (mappedBy = "city")
+    private Set<Address> addresses;
```

Comment:
This field looks like as redundant. You will not save addresses by City entity or load all by it. For that we will use separate service/repository method